dev/openresty/Dockerfile

```diff
@@ -25,16 +25,15 @@ RUN    apt-get -y install wget vim git libpq-dev tcpdump lua5.1 liblua5.1-dev un
 RUN  apt-get -y build-dep nginx \
   && apt-get -q -y clean && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
 RUN  wget https://openresty.org/download/openresty-1.21.4.2.tar.gz \
-  && wget https://github.com/Kong/kong/archive/master.tar.gz \
+  # && wget https://github.com/Kong/kong/archive/refs/tags/3.5.0.tar.gz -O kong.tar.gz \
```

Comment:
This shouldn't pin 3.5. Either we move with the latest `stable` tag if master is too aggressive or have separate release branches that are tied to Kong versions.

Reply:
Forgot to remove this commented line.